src/ui/components/ExplorerHome/ExplorerHome.js

```diff
@@ -154,17 +154,32 @@ const TIMEFRAME_OPTIONS: Array<{|
   +tabLabel: string,
   +tableLabel: string,
   +selector: (IntervalSequence) => Interval,
+  +chartSelector: (IntervalSequence) => Interval,
 |}> = deepFreeze([
   {
     tabLabel: "This Week",
     tableLabel: "This Week’s Activity",
     selector: (intervals) => intervals[intervals.length - 1],
+    chartSelector: (intervals) =>
+      intervals.length === 1
+        ? intervals[0]
+        : {
+            startTimeMs: intervals[intervals.length - 2].startTimeMs,
+            endTimeMs: intervals[intervals.length - 1].endTimeMs,
+          },
   },
   {
     tabLabel: "Last Week",
     tableLabel: "Last Week’s Activity",
     selector: (intervals) =>
       intervals.length === 1 ? intervals[0] : intervals[intervals.length - 2],
+    chartSelector: (intervals) =>
+      intervals.length <= 2
+        ? intervals[intervals.length - 1]
+        : {
+            startTimeMs: intervals[intervals.length - 3].startTimeMs,
+            endTimeMs: intervals[intervals.length - 2].endTimeMs,
+          },
```

Comment:
The way you have it, it would use this week's if there are 2 weeks. This changes makes it so that it uses last week's if there are two weeks.
```suggestion
    chartSelector: (intervals) =>
      intervals.length <= 2
        ? intervals[0]
        : {
            startTimeMs: intervals[intervals.length - 3].startTimeMs,
            endTimeMs: intervals[intervals.length - 2].endTimeMs,
          },
```

Reply:
Ah, of course! Thank you for catching my faulty math.